sql/core/benchmarks/FunctionBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,32 @@
+OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Mac OS X 10.16
```

Comment:
Can we create a benchmark via https://spark.apache.org/developer-tools.html#github-workflow-benchmarks?

Reply:
Sure let me try. DIdn't know this before.

Reply:
@HyukjinKwon Updated with benchmark results from GitHub Actions. One thing perhaps worth mentioning in the above doc is, when creating a new benchmark, the user needs to first include the benchmark result files in the PR. Otherwise the Github workflow will fail on the command `git diff --name-only`.

Reply:
Oh yeah. It should be fixed now at https://github.com/apache/spark/commit/be6ecb6d19528cc80e3c09e6127d2a89e5b4aa13 ...

Reply:
Oh, good point. I'll fix it soon. cc @byungsoo FYI who's working on updating the ML benchmark.

Reply:
@HyukjinKwon Sorry, I think you pinged a wrong person. :P I'm not a part of this project.

Reply:
Oops, I mean @byungsoo-oh :-). I made a fix here https://github.com/apache/spark/pull/32428 for the issue @sunchao

Reply:
Thanks @HyukjinKwon !